app/Http/Controllers/ProfileController.php

```diff
@@ -10,11 +10,45 @@
 
 class ProfileController extends Controller
 {
-    public function show($username)
+    public function show($userId)
     {
-        $user = User::where('name', $username)->firstOrFail();
-        $posts = Post::latest()->where('user_id', Auth::id())->get();
+        $user = User::where('id', $userId)->firstOrFail();
```

Comment:
```suggestion
        $user = User::firstOrFail($userId);
```
sửa những chỗ tương tự